test/e2e/storage/vsphere/vsphere_volume_master_restart.go

```diff
@@ -42,7 +42,7 @@ import (
 
 // waitForKubeletUp waits for the kubelet on the given host to be up.
 func waitForKubeletUp(ctx context.Context, host string) error {
-	cmd := "curl http://localhost:" + strconv.Itoa(ports.KubeletReadOnlyPort) + "/healthz"
+	cmd := "curl http://localhost:" + strconv.Itoa(ports.KubeletHealthzPort) + "/healthz"
```

Comment:
is this being exercised in presubmit?

Reply:
This may not be exercised in presubmit as `[Feature:vsphere][Serial][Disruptive]`.

Either port should work. Since read-only port will be disabled by default, alter it to use the health port.
```
[root@paco-centos-9 ~]# curl http://127.0.0.1:10248/healthz
ok[root@paco-centos-9 ~]# curl http://127.0.0.1:10255/healthz
ok
```